graylog2-server/src/main/java/org/graylog2/indexer/rotation/strategies/TimeBasedRotationStrategyConfig.java

```diff
@@ -35,18 +36,24 @@
     @JsonProperty("rotation_period")
     public abstract Period rotationPeriod();
 
+    @JsonProperty("elasticsearch_max_write_index_age")
```

Comment:
I think for this class we should name it `max_rotation_period` to be consistent with the `rotation_period` naming.

Reply:
fixed